packages/runtime/container-runtime/src/gc/gcConfigs.ts

```diff
@@ -158,7 +158,7 @@ export function generateGCConfigs(
 
 	// If version upgrade is not enabled, fall back to the stable GC version.
 	const gcVersionInEffect =
-		mc.config.getBoolean(gcVersionUpgradeToV3Key) === false
+		mc.config.getBoolean(gcVersionUpgradeToV2Key) === false
```

Comment:
At this point I'd rather rip out the config hook altogether.  We're not rolling that back, it's fine.